app/common/src/popup/Popup.js

```diff
@@ -22,7 +23,9 @@ export default class Popup extends React.Component<Props> {
       onBackdropPress={this.onClose}
     >
       <View style={styles.body}>
-        <ScrollView>{this.props.children}</ScrollView>
+        <ScrollView scrollEnabled={this.props.doScroll !== false}>
```

Comment:
See? Even `ScrollView` has the ability to disable scroll. This is even more strange since scrolling is what is it all about but it still makes sense. You could've used `scrollEnabled` property as well because it's more descriptive than `doScroll`. Property `somethingEnabled` implicates boolean value but `doSomething` implicates callback or command to do something...